src/lib.rs

```diff
@@ -108,23 +108,23 @@
 //! `.toml` file.
 //!
 //! Example:
-//!
-//! With a `[[bin]]` like:
+//! With the following code:
 //! ```rust,ignore
-//! fn main() {
-//!     println!("Hello world");
-//! }
+//! println!("{}", message);
 //! ```
 //!
-//! You can verify a code block like:
+//! You get the following:
 //! ~~~md
 //! ```console
-//! $ my-cmd
-//! Hello world
+//! $ my-cmd --print 'Hello World'
+//! Hello
 //!
 //! ```
```

Comment:
Please revert these changes; they are unrelated to the current change and, while there might be some discussion on iteration on it, I'd rather we decouple it from this specific PR.

Reply:
Dang, sorry. This and the imports have been unintentional, thanks for catching it. Fixed my rebase.